packages/@glimmer/validator/lib/utils.ts

```diff
@@ -0,0 +1,10 @@
+export type UnionToIntersection<U> = (U extends any ? (k: U) => void : never) extends ((
+  k: infer I
+) => void)
+  ? I
+  : never;
+
+export const symbol =
```

Comment:
Can you add a comment here about why this is duplicated (because we can't depend on `@glimmer/utils` without borking consumer types in `@glimmer/tracking`)